library/sha256.c

```diff
@@ -54,6 +54,7 @@
 #      if defined(__linux__)
```

Comment:
Please write a [changelog entry](https://github.com/Mbed-TLS/mbedtls/blob/development/ChangeLog.d/00README.md). Something like:

```
Bugfix
   * Fix a build error on [some Linux environments] when enabling ARMv8 SHA-2 acceleration. Bug reported and fix contributed by valord577.
```
where `[some Linux environments]` needs to be clarified (what's different between CentOS 7 and the versions we've tested — I'm not sure what they are but we mostly use Ubuntu?).